docs/troubleshooting-guide/proxies.md

```diff
@@ -0,0 +1,33 @@
+If you're experiencing problems where saving questions/dashboard sometimes fails or Metabase only loads a blank page, then it might be caused by a proxy.
```

Comment:
```suggestion
If you're experiencing problems where attempting to save a question or dashboard sometimes fails, or Metabase only loads a blank page, this might be caused by the use of a proxy.
```